As of 0.1.6, no mention of pipdeptree anywhere in the actual code.

--- a/.config/requirements.in
+++ b/.config/requirements.in
@@ -11,5 +11,4 @@
 mkdocstrings-python>=0.8.3
 mkdocstrings>=0.21.2
 pillow>=9.4.0
-pipdeptree>=2.4.0
 pymdown-extensions>=9.9.2
--- a/.config/requirements.txt
+++ b/.config/requirements.txt
@@ -39,7 +39,6 @@
 mkdocstrings-python==1.1.0
 packaging==23.1
 pillow==9.5.0
-pipdeptree==2.7.1
 pycparser==2.21
 pygments==2.15.1
 pymdown-extensions==10.0.1
